### PR TITLE
Show Annotations switch

### PR DESCRIPTION
**Pull request by PaulHax**
image

### CODE REVIEW

File: src/nrtk_explorer/app/transforms.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def delete_meta_state(old_ids, new_ids):
         )
 
         self.server.controller.add("on_server_ready")(self.on_server_ready)
```
Comment (jourdain) [inline src/nrtk_explorer/app/transforms.py:180]: `self.server.controller.on_server_ready.add(self.on_server_ready)` to be inline with the following line

File: src/nrtk_explorer/app/ui/image_list.py [hunk 2]
```diff
@@ -316,10 +350,10 @@ def __init__(self, on_scroll, on_hover, **kwargs):
                                         "Transformed: Detection Annotations",
                                         classes="text-center",
                                     )
-                                    ImageDetection(
+                                    ImageWithSpinner(
                                         identifier=("props.row.transformed",),
-                                        src=("get(props.row.transformed)",),
-                                        annotations=("props.row.transformedAnnotations",),
+                                        src="get(props.row.transformed)",
```
Comment (jourdain) [inline src/nrtk_explorer/app/ui/image_list.py:355]: `src=("get(props.row.transformed)",),`

File: src/nrtk_explorer/app/ui/image_list.py [hunk 3]
```diff
@@ -297,10 +331,10 @@ def __init__(self, on_scroll, on_hover, **kwargs):
                                         "Original: Detection Annotations",
                                         classes="text-center",
                                     )
-                                    ImageDetection(
+                                    ImageWithSpinner(
                                         identifier=("props.row.original",),
-                                        src=("props.row.original_src",),
-                                        annotations=("props.row.originalAnnotations",),
+                                        src="props.row.original_src",
```
Comment (jourdain) [inline src/nrtk_explorer/app/ui/image_list.py:336]: don't we want to have `:src="..."` or `v-bind:src="..."`? If so we should use the tuple notation

Comment (jourdain) [reply src/nrtk_explorer/app/ui/image_list.py:336]: never mind, you did it in the Python layer of ImageWithSpinner

File: src/nrtk_explorer/app/ui/image_list.py (hunk 2 above)
Comment (PaulHax) [reply src/nrtk_explorer/app/ui/image_list.py:355]: It was a string so I could check the value of transformed in the show spinner logic.  Better way?
```
            quasar.QInnerLoading(
                showing=(
                    # transformed image src props are refs, so check if ref is not set
                    f"(typeof {src} === 'object' && !{src}.value) || (show_annotations_on_images && !{annotations}.value)",
                )
            )
```

Comment (jourdain) [reply src/nrtk_explorer/app/ui/image_list.py:355]: the get() method return a ref. Editing showing is not a good option.

Comment (PaulHax) [reply src/nrtk_explorer/app/ui/image_list.py:355]: Shucks, I'm still not seeing how to write the `showing` expression.  I chaged it to: 
```
showing=(f"!{src} || (show_annotations_on_images && !{annotations}.value)",)`, 
```
but get JS syntax error:
```
{ showing: !('get(props.row.transformed_src).value',) || (show_annotations_on_images && !props.row.transformedAnnotations.value) }
```

Comment (jourdain) [reply src/nrtk_explorer/app/ui/image_list.py:355]: just make your src expect a tuple and just resolve it in showing=f"{src[0]} ..."

Comment (PaulHax) [reply src/nrtk_explorer/app/ui/image_list.py]: Ah, naturaly, thank you.

File: src/nrtk_explorer/app/ui/image_list.py [hunk 4]
```diff
@@ -297,10 +329,10 @@ def __init__(self, on_scroll, on_hover, **kwargs):
                                         "Original: Detection Annotations",
                                         classes="text-center",
                                     )
-                                    ImageDetection(
+                                    ImageWithSpinner(
                                         identifier=("props.row.original",),
                                         src=("props.row.original_src",),
-                                        annotations=("props.row.originalAnnotations",),
+                                        annotations="props.row.originalAnnotations",
```
Comment (jourdain) [inline src/nrtk_explorer/app/ui/image_list.py:335]: it is kind of weird that annotations behave differently than src... I would put tuples everywhere for consistency

Comment (PaulHax) [reply src/nrtk_explorer/app/ui/image_list.py]: ✅

**Review by jourdain — APPROVED** [review]

**Pull request merged**